CHANGELOG.md

```diff
@@ -28,6 +28,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Firecracker snapshots (up to Firecracker version v1.6.0) incompatible with
   the current Firecracker version.
 
+### Deprecated
+
+- Firecracker's `--start-time-cpu-us` parameter is deprecated and will be
+  removed in v2.0. It is used by the jailer to pass the value that should be
```

Comment:
nit: not sure it's necessary to promise to remove it in v2.0.

Reply:
How about `will be removed in v2.0 or later`?